.pre-commit-config.yaml

```diff
@@ -1,33 +1,9 @@
 repos:
-  # C++
+  # liniting c++ and python
   - repo: local
     hooks:
     - id: format
       name: format
       entry: bash ./scripts/format.sh
       language: system
       pass_filenames: false
-
-  # Python formatter
-  - repo: https://github.com/psf/black
-    rev: 22.10.0
-    hooks:
-      - id: black
-        language_version: python3
-        args: ['--line-length=105', '--skip-string-normalization']
-        files: ^(src/bindings/python/|scripts/)
-
-  # Python linter
-  - repo: https://github.com/pycqa/flake8
-    rev: 5.0.4
-    hooks:
-      - id: flake8
-        # E501: ignore line length (black auto-formats)
-        # E731: ignore replace-lambda-with-def warnings
-        # E203: ignore "whitespace before ':'"
-        args: [
-          '--max-line-length=105',
-          '--extend-ignore=E501,E731,E203',
-          '--extend-exclude=src/bindings/python/__init__.py'
-        ]
-        files: ^(src/bindings/python/|scripts/)
```

Comment:
each of these hooks is a 1-liner in the format script now, which is still called above so pre-commit still does the work if you like it that way